tools/nl/embeddings/sheets/embeddings_us_filtered.csv

```diff
@@ -201,7 +201,7 @@ Number,Name,Id,Description Strings,PaLM Generated Alternatives,ChatGPT descripti
-,Population: Black or African American Alone,Count_Person_BlackOrAfricanAmericanAlone,Number of people who are Black or African American Alone,population of black or african american alone; number of black or african american alone; number of people who are black or african american alone; count of black or african american alone; how many black or african american alone; number of people with race black or african american alone; how many black or african american alone live in;,,Number of People Who Are Black or African American Alone,,Number of people who are Black or African American Alone;Number of People Who Are Black or African American Alone;
+,Population: Black or African American Alone,Count_Person_BlackOrAfricanAmericanAlone,Number of people who are Black or African American Alone,population of black or african american alone; number of black or african american alone; number of people who are black or african american alone; count of black or african american alone; how many black or african american alone; number of people with race black or african american alone; how many black or african american alone live in;,"Black Population Alone,Population who are Black or African American Alone; Population who are Black or African American only;black people;Monoracial black population; how much of the population is black; african american population in; number of people who are black; count of black people in population; black and african american demographic",Number of People Who Are Black or African American Alone,,"Number of people who are Black or African American Alone;Black Population Alone,Population who are Black or African American Alone; Population who are Black or African American only;black people;Monoracial black population; how much of the population is black; african american population in; number of people who are black; count of black people in population; black and african american demographic;Number of People Who Are Black or African American Alone;"
```

Comment:
Any idea why these change?

Reply:
I suspect a change in the descriptions to the spreadsheet? seems like more were added?